packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -580,6 +584,7 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
         return;
       }
     },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I think it is preferable simply to include the ref in the dependency array, even though it will not change.